deploy/non-olm/v1.2.2/controller-4.yml

```diff
@@ -15,7 +15,5 @@ spec:
   mig_pod_limit: 100
   mig_namespace_limit: 10
 
-  mig_failure_rollback: false
-
```

Comment:
Why are we doing this on 1.2.2? That has been released upstream and downstream with this option. We should only be modifying latest and v1.3.0 if we're aiming to include it there as well.